README.md

```diff
@@ -50,22 +50,41 @@ Responsibilities and roles of the bridge:
 
 ## Usage
 
-### Install Dependencies
+There are two ways to deploy contracts: one option is to install and use NodeJS directly on the system,
```

Comment:
There are two ways to deploy contracts:

- install and use NodeJS
- use Docker to deploy

Reply:
Thanks. Done.